lib/services/registered_addons.pm

```diff
@@ -41,8 +41,8 @@ sub check_registered_system {
 sub check_registered_addons {
     my ($addonlist) = @_;
     $addonlist //= get_var('SCC_ADDONS');
```

Comment:
SCC_ADDONS really set the correct value?

Reply:
I have set bless list at $addonls = $addonls . ",base,desktop,sdk,lgm,python2,serverapp,wsm" if (is_sle('<15', get_var('HDDVERSION')) and is_sle('15+',.   So, it can automatic add those modules.